circle.yml

```diff
@@ -0,0 +1,8 @@
+machine:
+  node:
+    version: 5
+
+dependencies:
+  override:
+    - npm install
+    - ./node_modules/.bin/bower install
```

Comment:
Is this really the appropriate way to do such?  I don't think circle is putting the node_modules into the path.  So I'm unable to run `bower` alone; I must have the full path to it.